### PR TITLE
Adds more toy guns

### PR DESCRIPTION
**Pull request by Michiyamenotehifunana**
:cl: Toriate
add: Added foam Magrifles
add: Added foam Stealth Pistols
add: Added toy Hyper Burst Rifles
add: Added toy ray gun
tweak: foam X9s now available in autolathe
soundadd: added raygun noises
imageadd: added all the sprites for all those toys
/:cl:

The addition of these toys should scratch an itch for some people who want fun guns but without the whole killing, harming, or hurting people thing. The toy hyper burst rifle and toy ray guns are compatible with the laser tag armors and stuff.

All of the new toys can be made in the autolathe.

dreammaker_2017-10-06_15-57-22

dreammaker_2017-10-06_13-15-29

### CODE REVIEW

File: code/citadel/cit_guns.dm [hunk 1]
```diff
@@ -556,6 +592,53 @@
...
+	name = "toy hyper-burst launcher"
+	desc = "A toy laser with a unique beam shaping lens that projects harmless bolts capable of going through objects. Compatible with existing laser tag systems."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/magtag)
+	icon = 'icons/obj/guns/cit_guns.dmi'
+	icon_state = "toyburst"
+	clumsy_check = 0
+	needs_permit = 0
+	fire_delay = 40
+	weapon_weight = WEAPON_HEAVY
+	selfcharge = 1
```
Comment (silicons) [inline code/citadel/cit_guns.dm]: TRUE

File: code/citadel/cit_guns.dm [hunk 2]
```diff
@@ -556,6 +592,53 @@
...
+	e_cost = 1000
+	fire_sound = 'sound/weapons/magburst.ogg'
+
+/obj/item/gun/energy/laser/practice/hyperburst
+	name = "toy hyper-burst launcher"
+	desc = "A toy laser with a unique beam shaping lens that projects harmless bolts capable of going through objects. Compatible with existing laser tag systems."
+	ammo_type = list(/obj/item/ammo_casing/energy/laser/magtag)
+	icon = 'icons/obj/guns/cit_guns.dmi'
+	icon_state = "toyburst"
+	clumsy_check = 0
```
Comment (silicons) [inline code/citadel/cit_guns.dm]: FALSE

File: code/citadel/cit_guns.dm [hunk 3]
```diff
@@ -583,6 +666,40 @@
...
+///foam stealth pistol///
+
+/obj/item/gun/ballistic/automatic/toy/pistol/stealth
+	name = "foam force stealth pistol"
+	desc = "A small, easily concealable toy bullpup handgun. Ages 8 and up."
+	icon = 'icons/obj/guns/cit_guns.dmi'
+	icon_state = "foamsp"
+	w_class = WEIGHT_CLASS_SMALL
+	mag_type = /obj/item/ammo_box/magazine/toy/pistol
+	can_suppress = 0
```
Comment (silicons) [inline code/citadel/cit_guns.dm]: FALSE

File: code/citadel/cit_guns.dm [hunk 4]
```diff
@@ -583,6 +666,40 @@
...
+	fire_sound = 'sound/weapons/gunshot_silenced.ogg'
+	suppressed = 1
```
Comment (silicons) [inline code/citadel/cit_guns.dm]: TRUE

**Comment by Michiyamenotehifunana** [comment]
That should be all the booleans added in the PR. The rest of the booleans in the file will have to be fixed later on tho.

**Pull request merged**